pkg/api/volumes.go

```diff
@@ -61,11 +61,11 @@ func UpdateVolumeObject(target *common.VolumeObject, source *client.VolumesResou
 }
 
 // CreateVolume : creates a volume with the given name, capacity in the given pool
-func (client *Client) CreateVolume(name, size, pool, poolType string) (*common.VolumeObject, *common.ResponseStatus, error) {
+func (myclient *Client) CreateVolume(name, size, pool string) (*common.VolumeObject, *common.ResponseStatus, error) {
```

Comment:
Why myclient? Did you run into a name collision?

Reply:
Yes, that's a leftover from v1 of this PR. I needed access to a type defined in the client module but couldn't reach it due to the name being shadowed by the 'client' pointer. I could change it back to client now that I don't need to explicitly reference that type.

Reply:
Please do change it back to reduce the number of questions raised. Thank you.